### PR TITLE
Change default branch from master to main

### PR DESCRIPTION
**Pull request by jsignell**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@main
```
Comment (jacobtomlinson) [inline .github/workflows/ci.yaml:8]: These are branches on external repos, so are not under our control to change.

File: versioneer.py [hunk 2]
```diff
@@ -620,7 +620,7 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
         # refs/heads/ and refs/tags/ prefixes that would let us distinguish
         # between branches and tags. By ignoring refnames without digits, we
         # filter out many common branch names like "release" and
-        # "stabilization", as well as "HEAD" and "master".
+        # "stabilization", as well as "HEAD" and "main".
```
Comment (jacobtomlinson) [inline versioneer.py:623]: This is a comment made within the versioneer code. No harm in changing but I think it would invalidate the comment.

**Review by jacobtomlinson — COMMENTED** [review]
Thanks for the work here @jsignell. Couple of comments but happy to move forwards. CI failures are known issue and unrelated.

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (jsignell) [reply .github/workflows/ci.yaml:8]:
```suggestion
      - uses: actions/checkout@master
```

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -26,7 +26,7 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@main
```
Comment (jsignell) [inline .github/workflows/ci.yaml:29]:
```suggestion
      - uses: actions/checkout@master
```

File: versioneer.py (hunk 2 above)
Comment (jsignell) [reply versioneer.py:623]: I think the comment is still valid.

Comment (jacobtomlinson) [reply versioneer.py:623]: Oh yeah I guess so!

**Review by jsignell — COMMENTED** [review]
I think this one is good to merge whenever you are ready @jacobtomlinson

**Review by jrbourbeau — APPROVED** [review]
Thanks @jsignell! (test failures are unrelated)

**Comment by jacobtomlinson** [comment]
Thanks folks!

**Pull request merged**